src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -30,6 +30,9 @@ class FireCloudServiceActor extends HttpServiceActor {
     route(requestContext)
   }
 
+  // wraps route rejections in an ErrorReport
+  import org.broadinstitute.dsde.firecloud.model.ErrorReport.errorReportRejectionHandler
```

Comment:
This import covers all services in *production* code, but it must be added to each *test* individually.  I was unable to make that work as part of the new `ServiceSpec`.